rfcs/experimental/dynamic_selection/README.md

```diff
@@ -0,0 +1,516 @@
+| `template<Policy P, tyepname F, typename... Args> auto submit_and_wait(P&& p, F&& f, Args&&... args);` |  Invokes `f` with the unwrapped resource returned by `select(p, f, args…)` and `args`.And then waits on object returned by the `f`. May be implemented as `wait(p.submit(p.select(f, args…),f,args…))`. |
+| `template<typename P> auto get_submission_group(P&& p);` | Returns an object that has a member function `void wait()`. Calling this wait function blocks until all previous submissions to this policy are complete. |
+| `template<typename W> void unwrap(W&& w) noexcept;` | Returns `w.unwrap()` if available, otherwise returns `w`. |
+| `template<typename W> void wait(W&& w);` | Calls `w.wait()`. |
+| `template <typename S, typename Info> void report(S&& s, const Info& i);` | `S` is a *Selection*. Reports that event `i` has occurred if `s.report(i)` is available. |
+| `template <typename S, typename Info, typename Value> void report(S&& s, const Info& i, const Value& v); ` | `S` is a *Selection*. Reports a new value `v` for event `i` if `s.report(i, v)` is available. |
+
+### Deferred Initialization
+
+A call to `get_resources`, `select`, `submit` or `submit_and_wait` may initialize
+underlying state variables, including dynamic allocation. Initialization may throw `std::bad_alloc`.
+If `p` is a policy constructed with deferred initialization, calling these functions before 
+calling `initialize` will throw `std::logic_error`.
+
+<a id="execution_info_id"></a>
+## Execution Info
+
+Policies are informed of key events through the reporting of Execution Info.
+Most commonly, this reporting is done by a backend and is not visible to the
+end user.  However, developers that implement custom backends, or that
+manage work submission without using the `submit` or `submit_and_wait` functions,
+will need to report Execution Info to allow policies to work properly. There
+are currently three kinds of Execution Info that may be required by a Policy:
+
+| Execution Info    | Value Type | Description |
+| ----------------- | ---------- | ----------- |
+| `task_time`       | `uint64_t` | Clock ticks that elapsed between task submission and completion. |
+| `task_submission` | void       | The task has been submitted. |
+| `task_completion` | void       | The task is complete |
+
+| Info Traits* | Description |
+| ------- | ----------- |
+| `report_info<S,Info>::value`, `report_info_v<S,Info>` | 'true' if the *Selection* requires the event type to be reported |
+| `report_value<S,Info,V>::value`, `report_info_v<S,Info,V>` | `true` if the *Selection* requires the event value to be reported |
+
+* These traits are defined in `include/oneapi/dpl/internal/dynamic_selection_traits.h`
```

Comment:
The information about the header seems redundant.

Reply:
Will remove.